saleor/webhook/transport/synchronous/circuit_breaker/breaker_board.py

```diff
@@ -41,12 +89,21 @@ def register_error(self, app_id: int):
             f"{0}-{1}".format(app_id, "error"), self.ttl
         )
 
-        if errors >= self.failure_threshold:
+        open = False
+        if self.mode == BreakerMode.FIXED and errors >= self.failure_threshold:
+            open = True
+        elif self.mode == BreakerMode.PERCENTAGE:
+            total = self.storage.get_event_count("success") + errors
```

Comment:
There is a reason why we intended to keep track of `total` rather than `success`.

When registration of error or success event happens, the storage recomputes numer of events that are still alive (did not meet their TTL).

Fetching event count alone does give us that. Rather than expanding `get_event_count` (in my opinion) it'd be simpler to understand to increment `total` using already existing `register_event_returning_count` method when error **or** success occurs.